package-lock.json

```diff
@@ -2086,7 +2086,6 @@
         "anymatch": "^2.0.0",
         "async-each": "^1.0.1",
         "braces": "^2.3.2",
-        "fsevents": "^1.2.7",
```

Comment:
Drop this change.

Reply:
@YashJipkate, did you address this comment?

Reply:
I thought this was fixed. But it still does that. Reverted the change.